lib/broadside/deploy.rb

```diff
@@ -32,33 +29,27 @@ def full
     # The `yield` calls are a little weird but this was designed with an eye towards supporting other docker
     # based systems beyond ECS.  That day hasn't come yet, but we didn't think it was worth undoing the structure.
 
-    def rollback(count = @rollback)
+    def rollback(count)
       info "Rolling back #{count} release for #{family}..."
       yield
       info 'Rollback complete.'
     end
 
-    def scale
```

Comment:
Hmm nevermind i'm not sure that we'll ever get away from `Deploy` being an object seeing that there's a lot of benefit to having the `@target` attribute.

not sure how that plays ultimately with `Command` being a Module because all the commands also will want to have access to a `Target`.